apps/etterlatte-behandling/src/main/kotlin/oppgave/FristGaarUtJobb.kt

```diff
@@ -26,7 +26,7 @@ class FristGaarUtJobb(
     private val jobbNavn = this::class.simpleName
 
     override fun schedule(): Timer {
-        logger.info("$jobbNavn er satt til å starte $starttidspunkt med periode $periode")
+        logger.debug("{} er satt til å starte {} med periode {}", jobbNavn, starttidspunkt, periode)
```

Comment:
Litt smak og behag, men mye mer fan av hvordan det var tidligere med Kotlin sin string templating siden det er vesentlig lettere å lese.

Reply:
Er i grunn einig med det, men IntelliJ vil av ein eller annan grunn gjerne ha denne syntaksen for debug-statements. Mogleg det fins ein god grunn, mogleg det ikkje gjer 🤔